src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -653,30 +653,20 @@ public static double distanceSquared(final double[] x, final double[] y) {
      *
      * @param array
      * @param takeLog10OfOutput
-     * @param keepInLogSpace
+     * @param keepInLogSpace    if true, we don't normalize in the sense of probabilities summing to one but rather in the
+     *                          sense of numericla stability where we scale entries so that the largest is 0 in log space
      *
      * @return
      */
     public static double[] normalizeFromLog10(final double[] array, final boolean takeLog10OfOutput, final boolean keepInLogSpace) {
-        // for precision purposes, we need to add (or really subtract, since they're
-        // all negative) the largest value; also, we need to convert to normal-space.
-        double maxValue = arrayMax(array);
-
-        // we may decide to just normalize in log space without converting to linear space
         if (keepInLogSpace) {
+            double maxValue = arrayMax(array);
             return applyToArray(array, x -> x - maxValue);
```

Comment:
I though that variables used in lambdas needed to be final, I guess I was mistaken.